javatests/arcs/core/delegates/BUILD

```diff
@@ -0,0 +1,20 @@
+load(
+    "//third_party/java/arcs/build_defs:build_defs.bzl",
+    "arcs_kt_jvm_test_suite",
+)
+
+licenses(["notice"])
+
+package(default_visibility = ["//visibility:public"])
+
+arcs_kt_jvm_test_suite(
+    name = "delegates",
+    package = "arcs.core.delegates",
+    deps = [
+        "//src/wasm/kotlin/java/arcs",
+        "//third_party/java/junit",
+        "//third_party/java/truth",
+        "//third_party/kotlin/kotlinx/kotlinx_coroutines",
+        "//third_party/kotlin/kotlinx/kotlinx_coroutines:kotlinx_coroutines_test",
+    ],
+)
```

Comment:
I think this file should go in `src/wasm/kotlin/javatests` instead of `javatests/arcs/core/` right?

Reply:
That makes sense to me.

Reply:
Now at a completely code-generated solution so those tests went away.